Modules/_zoneinfo.c

```diff
@@ -2515,28 +2636,28 @@ new_weak_cache(void)
 }
 
 static int
-initialize_caches(void)
+initialize_caches(zoneinfo_state *state)
 {
     // TODO: Move to a PyModule_GetState / PEP 573 based caching system.
```

Comment:
Paul, did you have something specific in mind for this? For now, I've just moved everything as it is into the module state struct.

Reply:
See 23cbc43f372de581782d0476409708484132f59c